packages/eslint-plugin/package.json

```diff
@@ -59,6 +59,7 @@
     "@types/marked": "*",
     "@types/prettier": "*",
     "chalk": "^5.0.1",
+    "grapheme-splitter": "^1.0.4",
```

Comment:
We will need to add this as a peer dependency because package managers like yarn2 and pnpm will enforce that packages don't `require` unlisted dependencies.

I also think this will fix the failing integration test

Reply:
Ah right, thank you!

Reply:
Hmmrph, Yarn in the integration tests gives:

```plaintext
warning " > @typescript-eslint/eslint-plugin@5.46.1" has unmet peer dependency "grapheme-splitter@^1.0.4".
warning " > @typescript-eslint/eslint-plugin-tslint@5.46.1" has unmet peer dependency "grapheme-splitter@^1.0.4".
```

...moving to a dependency fixes it.